### DISCUSSION TITLE
[Feature Request]: Allow for setting bitrate for WiFi and 4G/5G Cellular connections separately

### DISCUSSION THREAD

**Discussion by NikolasDude** [ClassicOldSong/moonlight-android#75] [Ideas]
I think it would be quite useful to be able to set seperate bitrates for the type of internet connection you have.

For example, at home, I may set 50-100mbps as the connection is fast and stable, however when i'm out of town I would like to set a 20-30mbps bitrate to ensure the cellular connection does not lag or drop out.

**Comment by ClassicOldSong** [comment]
Cool idea, I sometimes want this as well lol

**Reply by NikolasDude** [reply]
Do you think this is something you might implement in the future? I would be very interested to see this feature come to life!

**Reply by ClassicOldSong** [reply]
Yes will do, but I have to think about how to make it a bit more user friendly.

**Reply by NikolasDude** [reply]
I'm glad to hear, sounds good!

**Reply by NikolasDude** [reply]
Would you mind shooting me a message / replying to this thread once this is implemented sometime down the road?
Considering I won't be updating my APK installations all too often.

Thanks! I am a fan of the improvements you have made to Moonlight / Sunshine so far

**Reply by ClassicOldSong** [reply]
Sure, you can also use Obtainium to track updates automatically. The link is in the bottom of the settings or you can find it in the README.

**Reply by NikolasDude** [reply]
Good to know, thanks for the tip

**Reply by NikolasDude** [reply]
Hey, I saw this got handled in a recent update, good work and thank you!

**Reply by ClassicOldSong** [reply]
Oh, it's you. It's been a long time...

(Well I'm not GLaDOS lol)

I tried all kinds of keywords, data, mobile, connection, cellular, just didn't though about 4G/5G 😂

**Reply by NikolasDude** [reply]
lol no worries, you tried!
I have been testing it out and it's been working great, no more bitrate warnings when I am on-the-go!

**Comment by invider42** [comment]
Hi this is a great functionality but I would know if it is possible to have multiple profiles of preferences for normal network (wifi home or other known slower wifi) or metered one (4g or 5g etc) ?
In these profiles it would be possible to ajust not only the bitrate, but the resolution, codec, etc.

**Reply by ClassicOldSong** [reply]
This can be considered when a UI refactor is done.
